ncm-gpfs/src/main/perl/gpfs.pm

```diff
@@ -434,6 +436,27 @@ sub get_cfg {
     }
     $gpfsconfigfh->close();
     $gpfsnodeconfigfh->close();
+
+    if ($tr->{keyData}) {
+        my $keydata = $tr->{keyData};
+        my $keyoutput = $self->runcurl($config, $tmp, $keydata);
```

Comment:
Feature request: can you please rely on `LWP::UserAgent` instead of shelling out to `curl`, which is quite cumbersome?